jobs/bump-jenkins-plugins.Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+import groovy.json.*
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+}
+
+properties([
+    pipelineTriggers([
+        // check once a month
+        pollSCM('H H 1 * *')
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+repo = "coreosbot-releng/fedora-coreos-pipeline"
+botCreds = "github-coreosbot-releng-token-username-password"
+pr_branch = "pluginsupdate"
+
+def getPluginLatestVersion(pluginsUpdate) {
```

Comment:
I think this probably would be easier to understand if we instead did a simpler `.split('/')` and then select the before last field.

Also let's include an example URL of what we're trying to parse.

And maybe let's call the signature e.g. `def getVersionFromPluginUrl(pluginUrl)`.